examples/jms-queue-message-receiver-sync-client-ack/jms-queue-message-receiver-sync-client-ack.bal

```diff
@@ -1,36 +1,39 @@
 import ballerina/jms;
 import ballerina/log;
 
-// Initialize a JMS connection with the provider
+// Initialize a JMS connection with the provider.
 jms:Connection jmsConnection = new({
         initialContextFactory:"wso2mbInitialContextFactory",
         providerUrl:"amqp://admin:admin@carbon/carbon?brokerlist='tcp://localhost:5672'"
     });
 
-// Initialize a JMS session on top of the created connection
+// Initialize a JMS session on top of the created connection.
 jms:Session jmsSession = new(jmsConnection, {
         acknowledgementMode:"CLIENT_ACKNOWLEDGE"
     });
 
-// Initialize a Queue sender on top of the the created sessions
+// Initialize a Queue sender on top of the the created session.
 endpoint jms:QueueReceiver queueReceiver {
     session:jmsSession,
     queueName:"MyQueue"
 };
 
 public function main(string[] args) {
-    // Receive a message from the JMS provider.
+    // Waits for 1 seconds for the message to be received by the JMS provider.
     var result = queueReceiver -> receive(timeoutInMilliSeconds = 1000);
 
     match result {
         jms:Message msg => {
+            // If the message is received, this block is executed. It checks for client acknowledgement.
```

Comment:
It is "not" checking for client acknowledgement. It is just acknowledge the message.